src/main/archetype/ui.apps/src/main/content/META-INF/vault/filter.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
+    <filter root="/apps/${appId}"/>

```

Comment:
This is way too wide and overlaps the filter in `ui.config`. Also, should not be needed because of https://github.com/adobe/aem-project-archetype/blob/master/src/main/archetype/ui.apps.structure/pom.xml#L36

Reply:
your right the warning for this one is actually on info level